<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Spectrum</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>671</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="freq1">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="freq2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="freq3">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="freq4">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="freq5">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="freq6">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="freq7">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="freq8">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="freq9">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="freq10">
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>            0.1                0.2                0.3                0.4                0.5                0.6               0.7                 0.8               0.9               1.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FileUrl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadFile">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartSpectrum">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
